libs/libyang/Makefile

```diff
@@ -55,20 +55,11 @@ define Package/libyang/description
 endef
 
 CMAKE_OPTIONS += \
-	-DENABLE_LYD_PRIV:BOOL=ON \
-	-DCMAKE_BUILD_TYPE:String="Release" \
-	-DGEN_LANGUAGE_BINDINGS=ON \
-	-DGEN_PYTHON_BINDINGS=OFF
+	-DCMAKE_BUILD_TYPE:String="Release"
```

Comment:
This is default: https://github.com/openwrt/openwrt/blob/master/include/cmake.mk#L94